subhub/tests/unit/test_tracing.py

```diff
@@ -0,0 +1,31 @@
+import functools
+
+from subhub.tracing import timed
+from random import randint
+
+
+@timed
+def generate_random_value():
+    return randint(0, 9)
+
+
+@timed
+def generate_wrapped_random_value():
+    return __wrapped(randint(0, 9))
+
+
+@timed
+def __wrapped(functor):
+    @functools.wraps(functor)
+    def call(*args, **kwargs):
+        return functor(*args, **kwargs)
+
```

Comment:
black line, probably not needed

Reply:
Checking now to see if this pleases the black formatter.

Reply:
Every time I take it away, black wants to put it back...